src/patternfly/components/Accordion/accordion.scss

```diff
@@ -84,25 +82,25 @@
     background-color: var(--pf-c-accordion__toggle--hover--BackgroundColor);
 
     .pf-c-accordion__toggle-text {
-      color: var(--pf-c-accordion__toggle-text--hover--Color);
+      color: var(--pf-c-accordion__toggle--hover__toggle-text--Color);
     }
   }
 
   &:active {
     background-color: var(--pf-c-accordion__toggle--active--BackgroundColor);
 
     .pf-c-accordion__toggle-text {
-      font-weight: var(--pf-c-accordion__toggle-text--active--FontWeight);
-      color: var(--pf-c-accordion__toggle-text--active--Color);
+      font-weight: var(--pf-c-accordion__toggle--active__toggle-text--FontWeight);
+      color: var(--pf-c-accordion__toggle--active__toggle-text--Color);
     }
   }
 
   &:focus {
```

Comment:
Oh sorry, I missed this before. Can you move the `:active` selector & styles after `:focus`? Otherwise `:focus` will always override `:active`